python-sdk/example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -86,19 +88,20 @@ def example_amazon_s3_snowflake_transform():
 
     cleaned_data = clean_data(combined_data)
     # [START dataframe_example_2]
+    snowflake_output_table = Table(
+        name="aggregated_adoptions_" + str(int(time.time())),
+        metadata=Metadata(
+            schema=os.environ["SNOWFLAKE_SCHEMA"],
+            database=os.environ["SNOWFLAKE_DATABASE"],
+        ),
+        conn_id="snowflake_conn",
+        temp=True,
+    )
     aggregate_data(
         cleaned_data,
-        output_table=Table(
-            name="aggregated_adoptions_" + str(int(time.time())),
-            metadata=Metadata(
-                schema=os.environ["SNOWFLAKE_SCHEMA"],
-                database=os.environ["SNOWFLAKE_DATABASE"],
-            ),
-            conn_id="snowflake_conn",
-        ),
+        output_table=snowflake_output_table,
     )
     # [END dataframe_example_2]
-    aql.cleanup()
```

Comment:
@rajaths010494 don't we need that line?

Reply:
yes we need it.  I missed this when resolving conflicts.

Reply:
added it back